web/packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -358,12 +358,12 @@ exports[`list of all events 1`] = `
           </strong>
            - 
           <strong>
-            167
+            172
```

Comment:
I'm not sure why the snapshots tests are failing, it appears that you did run `yarn test-update-snapshot` in the latest commit.

Reply:
I'll do that, thanks for catching that!